src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -156,10 +190,10 @@ public void toModelType_invalidStatus_throwsIllegalValueException() {
     public void toModelType_nullStatus_throwsIllegalValueException() {
         JsonAdaptedPerson person =
                 new JsonAdaptedPerson(VALID_NRIC, VALID_NAME, VALID_PHONE,
-                        VALID_ADDRESS, VALID_DOB, VALID_SEX, null);
+                        VALID_ADDRESS, VALID_DOB, VALID_SEX, null, VALID_EMAIL, VALID_COUNTRY,
+                        VALID_ALLERGIES, VALID_BLOODTYPE, VALID_CONDITION, VALID_DOA, VALID_DIAGNOSIS,
+                        VALID_SYMPTOM);
         String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, Address.class.getSimpleName());
         assertThrows(IllegalValueException.class, expectedMessage, person::toModelType);
     }
-
-
 }
```

Comment:
It would be nice to have invalid tests for optional fields too... but let's do it in v1.3.